### PR TITLE
Bl 637 fe combine components into main container

### PR DESCRIPTION
**Pull request by Joshmclemore**
## Description

Made navigation functional and placed "page not found" components where navigation is not yet possible

#### Video Link

https://www.loom.com/share/da21c0ff7b47423687ddebf587217b9e

#### Jira Link

https://bloomtechlabs.atlassian.net/browse/BL-637

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ ___Add your PR description here, deleting this example___
...
-#### Trello Link
+#### Jira Link
 
-___Paste your trello card link here, deleting this example___
+___Paste your jira ticket link here, deleting this example___
 
 (Example below):
 
-<blockquote class="trello-card"><a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;JfipwNkb&#x2F;265-fix-refactor-sidebar-to-have-icons-representing-each-element-in-the-sidebar">Fix: Refactor sidebar to have icons representing each element in the sidebar</a></blockquote>
+https://bloomtechlabs.atlassian.net/browse/BL-637
```
Comment (attaris978) [inline pull_request_template.md:15]: Are we supposed to be updating the template every time we submit a pull request? My assumption has been that we leave the template and fill out the form on github for each pull request.

File: src/components/Layout/NavBar.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ const NavBar = () => {
         color: '#3E2B2F',
       }}
       defaultSelectedKeys={['ab1']}
-      defaultOpenKeys={['ab']}
+      defaultOpenKeys={['']}
```
Comment (attaris978) [inline src/components/Layout/NavBar.js:21]: Setting it to a default closed state -- excellent.

File: src/components/Layout/items.js [hunk 3]
```diff
@@ -1,7 +1,4 @@
-// NOTE: I pulled the routes from HRF's website
-// because we don't have the same routes set up,
-// they currently link to mostly non-existent sites
-// e.g. there is no "/about"
+// links are absolute and will need to be changed if implemented on the main site.
```
Comment (attaris978) [inline src/components/Layout/items.js:1]: Good insight for those-that-come-after.

File: src/components/Layout/items.js [hunk 4]
```diff
@@ -372,13 +459,22 @@ const items = [
...
+      <a href="/heatmap" title="heatmap">
+        Heatmap
+      </a>,
+      'data4'
+    ),
   ]),
 ];
 export default items;
+
+// <Route path="/example-list" component={ExampleListPage} />
```
Comment (attaris978) [inline src/components/Layout/items.js:479]: These are an accidental paste, I think -- the Routes you removed from the index.jsx file.

File: src/index.jsx [hunk 5]
```diff
@@ -42,8 +40,7 @@ function App() {
           <Switch>
             <Route path="/" exact component={LandingPage} />
             <Route path="/table" component={TablePage} />
-            <Route path="/example-list" component={ExampleListPage} />
-            <Route path="/datavis" component={ExampleDataViz} />
+            <Route path="/heatmap" component={NotFoundPage} />
```
Comment (attaris978) [inline src/index.jsx:43]: I like how this enables future components to be implemented in the menu before being finished -- it makes the site feel more complete without it yet being completed.

**Review by attaris978 — CHANGES_REQUESTED** [review]
Looks great -- just need to remove those mysteriously-transfered Route components in the items.js file.

**Review by attaris978 — CHANGES_REQUESTED** [review]
Eek -- also, anything containing JSX should have the extension .jsx instead of .js.

**Review by Ricardorouge — APPROVED** [review]

**Review by attaris978 — APPROVED** [review]
💥

**Review by Ricardorouge — APPROVED** [review]

**Review by VWashingtonCoder — APPROVED** [review]
This looks like a pretty clean link-up for the Navbar. I find it useful that each one is fully linked to its representative webpage as well as the table hookup. The not found page link for the Heatmap is also a nice touch.

**Review by AlexiusThomas — COMMENTED** [review]
The explanation of the raw data being imported was very clear and well organized.

**Pull request merged**